internal/provider/transfer_resource_test.go

```diff
@@ -57,16 +57,6 @@ func TestAccTransferResource(t *testing.T) {
 											exclude = ["c2"]
 										}
 									}
-								},
```

Comment:
May you unskip this tests? It disabled by lines:
```go
	return
	//nolint:govet
```
Just delete them and let's see that everything is all right with your changes.